src/styles/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  padding: 16px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  width: calc(200px - 2 * 16px - 2px);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container-image {
+  margin-top: 16px;
+  margin-bottom: 36px;
+  margin-left: 3px;
```

Comment:
You can use shorthand property for that
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#syntax